.vscode/launch.json

```diff
@@ -21,7 +21,7 @@
       ],
       "env": {
         "QUARTO_BIN_PATH": "${workspaceFolder}/package/dist/bin",
-        "QUARTO_SHARE_PATH": "${workspaceFolder}/src/resources",
```

Comment:
We have code in the field that relies on `QUARTO_SHARE_PATH` (IDE uses it for crossref indexing). It's fine to change but we should just make sure that the old environment variable is still respected.

Reply:
Is QUARTO_SHARE_DIR also in use? I have no preference, I just didn't like that we had two variables for the same thing.

Reply:
Perhaps we use `QUARTO_SHARE_PATH` in the product and `QUARTO_SHARE_DIR` in some configure scripts? I don't know. @dragonstyle Could you provide clarity here?

Reply:
`QUARTO_SHARE_DIR` was used in configuration to provide the simple name of the share directory (e.g. `share`). `QUARTO_SHARE_PATH` was used in the bash script to contain the complete path to the share directory.

We followed this general pattern (`*_DIR` = just a simple name, `*_PATH` = an actual path) throughout configuration (or at least we generally should've).